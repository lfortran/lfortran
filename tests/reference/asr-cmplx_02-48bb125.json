{
    "basename": "asr-cmplx_02-48bb125",
    "cmd": "lfortran --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/errors/cmplx_02.f90",
    "infile_hash": "4bc293fe7261129d487b3058234cb9ec644c7d91a6e984cb0984a29c",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-cmplx_02-48bb125.stderr",
    "stderr_hash": "6903dfab9abef1f469ce7a2042a3f733d4feae49a578327e7bfdcff4",
    "returncode": 2
}