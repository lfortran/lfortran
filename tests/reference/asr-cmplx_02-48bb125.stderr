semantic error: Expected a real constant for the real part
 --> tests/errors/cmplx_02.f90:2:13
  |
2 |     print*, cmplx((real(1, kind=4), 0.00000000), kind=8)
  |             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ 
